cypress/e2e/po/pages/explorer/cronjob.po.ts

```diff
@@ -0,0 +1,53 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
+
+class CronjobDetailPagePo extends PagePo {
+  private static createPath(clusterId: string, id: string) {
+    return `/c/${ clusterId }/explorer/batch.job/${ id }`;
+  }
+
+  goTo(clusterId: string, id:string): Cypress.Chainable<Cypress.AUTWindow> {
+    return super.goTo(CronjobDetailPagePo.createPath(clusterId, id));
+  }
+
+  constructor(clusterId = 'local', id: string) {
+    super(CronjobDetailPagePo.createPath(clusterId, id));
+  }
+
+  successfulCountCheck(countExpected: number, timeout = 15000) {
+    // let's give it a buffer for the job to finish running
+    cy.wait(timeout); // eslint-disable-line cypress/no-unnecessary-waiting
```

Comment:
you should be able to extend the time `get` allows for an element to appear by passing in args to cy.get (check out usages of MEDIUM_TIMEOUT_OPT, LONG_TIMEOUT_OPT, EXTRA_LONG_TIMEOUT_OPT)